### ISSUE TITLE
Exception: Unsupported/unrecognized mimetype: audio/aac

### ISSUE THREAD

**Issue by chriso0710** [jlguardi/yowsup#72]
Hi all,

I am referring to
https://github.com/tgalal/yowsup/issues/1433
and am getting the exact same error. getExtension throws an exception with audio media.

I am using @jlguardi's latest master branch.
The mimetype is included in yowsup/common/mime.types, but maybe is not used...? 
Another python package problem?

Regards
Christian

```
12:56:48 worker.1 |   File "/Users/co/projects/waposter/layer.py", line 66, in onMessage
12:56:48 worker.1 |     self.getMediaMessageBody(message)
12:56:48 worker.1 |   File "/Users/co/projects/waposter/layer.py", line 90, in getMediaMessageBody
12:56:48 worker.1 |     url = self.getDownloadableMediaMessageBody(message)
12:56:48 worker.1 |   File "/Users/co/projects/waposter/layer.py", line 101, in getDownloadableMediaMessageBody
12:56:48 worker.1 |     k.key = "%s%s"%(message.getId(),message.getExtension())
12:56:48 worker.1 |   File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/yowsup/layers/protocol_media/protocolentities/message_media_downloadable.py", line 78, in getExtension
12:56:48 worker.1 |     return MimeTools.getExtension(self.mimeType)
12:56:48 worker.1 |   File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/yowsup/common/tools.py", line 201, in getExtension
12:56:48 worker.1 |     raise Exception("Unsupported/unrecognized mimetype: "+mimetype);
12:56:48 worker.1 | Exception: Unsupported/unrecognized mimetype: audio/aac
```

**Comment by jlguardi** [comment]
Some hours ago I've tried to fix it. Are you updated?

**Comment by chriso0710** [comment]
No, sorry, did not pull the fix from today. 
I have no time for this right now, but I will test next week and add a comment here.

**Comment by jlguardi** [comment]
By now, I'll close this issue due to I think it's fixed with #70. Otherwise, reopen it again.

**Comment by chriso0710** [comment]
Yes, looking good, thank you.
